crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -120,13 +132,21 @@ pub enum TokenOwnerFindingStrategy {
 
     /// Use the Ethplorer token holder API.
     Ethplorer,
+
+    /// Use list provided by the external solver team - SeaSolver
+    SeaSolver,
 }
 
 impl TokenOwnerFindingStrategy {
     /// Returns the default set of token owner finding strategies.
     pub fn defaults_for_chain(chain_id: u64) -> &'static [Self] {
         match chain_id {
-            1 => &[Self::Liquidity, Self::Blockscout, Self::Ethplorer],
+            1 => &[
+                Self::Liquidity,
+                Self::Blockscout,
+                Self::Ethplorer,
+                Self::SeaSolver,
```

Comment:
I would not enable this by default in code. It seems very specific to that external solver, and, in theory, we only would want to enable it when running with that external solver.